src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -397,6 +397,29 @@ export default defineComponent({
       return this.playlistIdTypePairFinal?.playlistItemId
     },
 
+    quickBookmarkPlaylistId() {
+      return this.$store.getters.getQuickBookmarkTargetPlaylistId
+    },
+    quickBookmarkPlaylist() {
+      return this.$store.getters.getPlaylist(this.quickBookmarkPlaylistId)
+    },
+    isQuickBookmarkEnabled() {
+      return this.quickBookmarkPlaylist != null
+    },
+    isInQuickBookmarkPlaylist: function () {
+      if (!this.isQuickBookmarkEnabled) { return false }
+
+      return this.quickBookmarkPlaylist.videos.some((video) => {
+        return video.videoId === this.id
+      })
+    },
+    quickBookmarkIconText: function () {
+      return this.isInQuickBookmarkPlaylist ? this.$t('User Playlists.Remove from Favorites') : this.$t('User Playlists.Add to Favorites')
```

Comment:
**issue (blocking):** Playlist is not necessarily "Favorites". Suggestion: Use `playlistName`. E.g., `Remove from ${playlistName}`.